tardis/plasma/properties/ion_population.py

```diff
@@ -406,6 +400,12 @@ class IonNumberDensityHeNLTE(ProcessingPlasmaProperty):
     for the electron densities are not similar to within the threshold
     value, a new guess for the value of the electron density is chosen
     and the process is repeated.
+    
+    Attributes
+    ----------
+    ion_number_density : Pandas DataFrame, dtype float
+                         Index atom number, ion number. Columns zones.
```

Comment:
There is extra ident here

Reply:
That one should be fixed as I remember fixing it half an hour ago.